client/webserver/site/src/js/register.js

```diff
@@ -165,8 +165,8 @@ export default class RegistrationPage extends BasePage {
     this.fee = res.xc.feeAsset.amount
 
     page.feeDisplay.textContent = Doc.formatCoinValue(this.fee / 1e8)
-    const dcrAsset = res.xc.assets['42']
-    if (dcrAsset) page.dexDCRLotSize.textContent = Doc.formatCoinValue(dcrAsset.lotSize / 1e8)
+    // const dcrAsset = res.xc.assets['42']
+    page.dexDCRLotSize.textContent = '<depends on market>' // Doc.formatCoinValue(dcrAsset.lotSize / 1e8)
```

Comment:
Open for suggestions on how to handle this. Perhaps changing the text to something like "The server has markets with DCR as the base asset with the following lot sizes: ..."  Not sure how to cleanly implement or if there's a better way to just summarize *all* of the server's markets in a table for the user to decide.

Reply:
A small table does sound pretty good. It's not going to scale that well to a DEX with lots of DCR (or other reg asset) markets, but we can worry about that when we have more than 4 assets.

@chappjc, if you don't want to do the html, feel free to use a simpler solution of your choosing in the interim and we can open an issue.

Reply:
Went with this lot size for whatever DCR base market, preferring DCR_BTC, and a basic display:

![image](https://user-images.githubusercontent.com/9373513/124677158-7501c900-de85-11eb-8e8d-54c17f412d4a.png)